cedar-java-hello-world/settings.gradle

```diff
@@ -2,4 +2,3 @@
 // https://docs.gradle.org/current/userguide/organizing_gradle_projects.html#always_define_a_settings_file
 rootProject.name = 'CedarJavaHelloWorld'
 
-includeBuild("cedar-java/CedarJava")
```

Comment:
Should we remove corresponding files?

Reply:
Not sure I understand. AFAIK they aren't checked into git.

Reply:
how were users getting the `cedar-java` files?  if they weren't checked in, were there some README instructions somewhere, that now need to be changed since they don't need the `CedarJava` files?

Reply:
The ci.yml pulls both `CedarJava` and `CedarJavaFFI` (since they're in the same repo). We still need `CedarJavaFFI` so cloning `cedar-java` is still a step, we just don't use the `CedarJava` part of `cedar-java` anymore.

Reply:
I'll update the instructions once the CedarJava build is unblocked and users can download the `.dll` or `.so` files from github.